features/object-position.yml

```diff
@@ -3,5 +3,8 @@ description: The `object-position` CSS property places images, videos, and other
 spec: https://drafts.csswg.org/css-images-3/#the-object-position
 # object-fit on caniuse includes both object-fit and object-position.
 # `caniuse: object-fit` is set in object-fit.yml
+status:
+  compute_from: css.properties.object-position
```

Comment:
The description says "images, videos, and other replaced elements", but now we're ignoring iframes, which is also a replaced element. So I think we should do something like this: call out the iframe exception in the description, and add a TODO comment about removing the callout once iframes are taken into account.

Reply:
I really dislike the way this information has been represented in BCD. I think the addition of the `applies_to_iframe_elements` key ought to be reverted (and left a comment saying so). Until https://github.com/mdn/browser-compat-data/issues/23586 is resolved, I'd suggest:

- Undoing all the changes in this feature so far
- Adding a comment mentioning `css.properties.object-position.applies_to_iframe_elements` and https://github.com/mdn/browser-compat-data/issues/23586 and saying that the compat data for which elements `object-position` works with is unclear and needs to be sorted out upstream.

Reply:
Updated in e95bb194